docs/README.md

```diff
@@ -70,41 +77,85 @@ root = "./app"
 
 ### Third-party modules
 
-Third-party modules are external Python packages that are not part of the standard library or your own project’s code. These are typically installed using a package manager like uv or pip and include libraries such as `requests`, `numpy`, or `django`.
+Third-party modules are Python packages that are not part of your project or the standard library.
+These are usually dependencies of your project, and are installed using a package manager like uv or
+pip. Common examples include libraries such as `requests`, `numpy`, or `django`.
+
+ty searches for third-party modules in the configured [Python environment](#python-environment).
+
+### Python environment
+
+ty only supports automatic discovery of virtual environments at this time.
+
+First, ty checks for an active environment using the `VIRTUAL_ENV` environment variable. If not set,
+ty will search for a `.venv` folder in the project root or working directory.
 
-ty searches third-party modules in your project's virtual environment. By default, it looks for a virtual environment in a `.venv` folder located at the project root. If the `VIRTUAL_ENV` environment variable is set, ty will use the path specified there instead. If your project uses a different location for your virtual environment, specify the location by setting the [`environment.python`](./reference/configuration.md#python) configuration or [`--python`](./reference/cli.md#ty-check--python) CLI option.
+The Python environment may be explicitly specified using the
+[`environment.python`](./reference/configuration.md#python) setting or
+[`--python`](./reference/cli.md#ty-check--python) flag.
+
+When setting the environment explicitly, non-virtual environments can be provided.
 
 ## Python version
 
-The supported Python syntax and standard library functions differ between Python versions. For example, Python 3.10 introduced support for `match` statements and the `sys.stdlib_module_names` symbol. While these features enhance the language, using them in a project targeting an older Python version can lead to compatibility issues.
-ty helps you avoid such compatibility issues by checking your code against the Python version your project targets, and flagging any use of features or standard library symbols not available in that version.
+The Python version affects allowed syntax, type definitions of the standard library, and type
+definitions of first- and third-party modules that are conditional on the Python version.
+
+For example, Python 3.10 introduced support for `match` statements and added the the
+`sys.stdlib_module_names` symbol. However, if your project also supports Python 3.9, you cannot use
+these new features.
```

Comment:
````suggestion
`sys.stdlib_module_names` symbol to the standard library. However, if your project also supports
Python 3.9, you cannot use these new features unless they are inside an `if sys.version_info >= (3, 10)` branch:

```python
import sys

if sys.version_info >= (3, 10):
    print(sys.stdlib_module_names)  # okay whatever your python-version is set to, because of the `sys.version_info` condition
else:
    print(sys.stdlib_module_names)  # ty will emit an error here if the configured `python-version` is <3.10
```
````